app/api/migrations/migrations/36-populate-mimetype-on-attachments/specs/36-populate-mimetype-on-attachments.spec.js

```diff
@@ -0,0 +1,136 @@
+import testingDB from 'api/utils/testing_db';
+import request from 'shared/JSONRequest';
+import * as attachmentMethods from 'api/files/filesystem';
+import childProcess from 'child_process';
+import migration from '../index.js';
+import fixtures from './fixtures.js';
+
+describe('migration populate-mimetype-on-attachments', () => {
```

Comment:
I think this test suite needs to include at least one test with mixed cases.

Reply:
Added one more test...